CryptoPrices/Sources/Home/Sources/Home/WalletStatisticSectionView/WalletStatisticSection.swift

```diff
@@ -46,7 +46,7 @@ private extension WalletStatisticSection {
                     .foregroundColor(Colors.lightSilver.swiftUIColor)
                     .font(Fonts.Inter.medium.textStyle(.callout))
 
-                Text("$7,273,291") // TODO: Handle showing this number with device locale
+                Text(7_273_291, format: .dollarCurrency) // TODO: Handle showing this number with device locale
```

Comment:
Since we have setup the `DollarCurrencyFormatStyle` already, can we remove the TODO here? 🤔

Reply:
Yeah, it should be replaced with another one like `TODO: Remove hard-coded data` instead 👍